de.schmidhuberj.DieBahn.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "de.schmidhuberj.DieBahn",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "diebahn",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "env": {
+            "CARGO_HOME": "/run/build/diebahn/cargo",
+            "RUST_BACKTRACE": "1"
+        }
+    },
+    "modules": [
+        "gtk.json",
+        "libadwaita.json",
```

Comment:
I'm not a big fan of splitting modules to separate files but it's just a nitpick.